tests/test_reclaim_swap.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+'''
+Tests eos-reclaim-swap.
+'''
+
+import os
+import subprocess
+import tempfile
+
+from .util import (
+    BaseTestCase,
+    system_script
+)
+
+
+EOS_RECLAIM_SWAP = system_script('eos-reclaim-swap')
+DATA_DIR = './data/reclaim-swap'
+
+class TestReclaimSwap(BaseTestCase):
+
+    def get_data(self, basename):
+        '''
+        Return contents of file at DATA_DIR/basename.
+        '''
+        path = os.path.join(DATA_DIR, basename)
+        with open(path, 'rb') as f:
+            data = f.read()
+        return data
+
+
+    def dup_image(self, testname):
+        '''
+        Return a copy of DATA_DIR/testname.img in the current working dir.
+        '''
+        basename = testname + '.img'
+        path = os.path.join(DATA_DIR, basename)
+        with open(path, 'rb') as f:
+            data = f.read()
+        img = open(basename, 'wb')
+        img.write(data)
+        img.flush()
+        return img
```

Comment:
I had something similar initially using tempfile (I even left the import behind), but it does not work because the file name is part of the expected resulting partition table.

But I realized I need to use `@contextlib.contextmanager` and `yield` to have img be automatically closed when exiting the `with self.dup_image(test) as img:`, so I'm taking that. (haven't used `with` before this).